packages/devextreme-scss/scss/widgets/generic/chat/_colors.scss

```diff
@@ -27,6 +28,7 @@ $chat-messagelist-empty-prompt-color: null !default;
   $chat-messagelist-empty-icon-background-color: darken($base-bg, 7.84) !default; // #EBEBEB 
   $chat-messagelist-empty-message-color: darken($base-text-color, 6.67) !default; // #222222
   $chat-messagelist-empty-prompt-color: lighten($base-label-color, 38.43) !default; // #959595
+  $chat-messagelist-day-header-color: lighten($base-icon-color, 46.67) !default; // #AAAAAA
```

Comment:
I assume that it will be unexpected for the user when, after changing the base-icon-color, the text color of the header in the chat also changes. Let's fix this. The variable should at least depend on the text color.

Reply:
Kindly fix it in other places

Reply:
As far as i remember base-icon-color equal to base-text-color, so it should not be problematic to change this

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925